### PR TITLE
Add BananaPi BPI-PicoW-S3 board.

### PR DESCRIPTION
**Pull request by Wind-stormger**
Support double reset to tinyUF2.
PID from Espressif.

### CODE REVIEW

File: ports/espressif/boards/bpi_picow_s3/board.h [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+// USB UF2
+//--------------------------------------------------------------------+
+
+#define USB_VID                  0x303A
+#define USB_PID                  0x812D
+#define USB_MANUFACTURER         "BananaPi"
+#define USB_PRODUCT              "BPI-PicoW-S3"
+
+#define UF2_PRODUCT_NAME         USB_MANUFACTURER " " USB_PRODUCT
+#define UF2_BOARD_ID             "BPI-PicoW-S3"
```
Comment (hathach) [inline ports/espressif/boards/bpi_picow_s3/board.h:64]: board id need to follow uf2 specs https://github.com/microsoft/uf2#files-exposed-by-bootloaders

**Review by hathach — CHANGES_REQUESTED** [review]

File: ports/espressif/boards/bpi_picow_s3/board.h (hunk 1 above)
Comment (Wind-stormger) [reply ports/espressif/boards/bpi_picow_s3/board.h]: Thanks for the review, fixed.

**Review by hathach — CHANGES_REQUESTED** [review]
one more thing, you need to add this board to ci so that its binary is available in the release https://github.com/adafruit/tinyuf2/blob/master/.github/workflows/build_esp32.yml

**Comment by Wind-stormger** [comment]
Thanks for the reminder, added.

**Review by hathach — APPROVED** [review]
look good now

**Pull request merged**